overlays/crypto/blst.nix

```
@@ -10,10 +11,37 @@ stdenv.mkDerivation rec {
     sha256 = "01m28xxmm1x7riwyax7v9rycwl5isi06h2b2hl4gxnnylkayisn5";
   };
 
-  buildPhase = "./build.sh";
+  buildPhase = ''
+    ./build.sh ${lib.optionalString stdenv.targetPlatform.isWindows "flavour=mingw64"}
+  '' + ''
+    ./build.sh -shared ${lib.optionalString stdenv.targetPlatform.isWindows "flavour=mingw64"}
+  '';
   installPhase = ''
     mkdir -p $out/lib
-    cp ./libblst.a $out/lib/
+    for lib in libblst.{a,so,dylib}; do
+      if [ -f $lib ]; then
+        cp $lib $out/lib/
+      fi
+    done
+    for lib in blst.dll; do
+      if [ -f $lib ]; then
+        mkdir -p $out/bin
+        cp $lib $out/bin/
+      fi
+    done
+  '' + ''
+    mkdir -p $out/lib/pkgconfig
+    cat <<EOF > $out/lib/pkgconfig/libblst.pc
+    prefix=$out
+    exec_prefix=''\\''${prefix}
+    libdir=$out/lib
+
+    Name: libsodium
```

Should this be `blst` instead of `sodium`?
```suggestion
    Name: libblst
```